share/spice/glassdoor/glassdoor.js

```diff
@@ -0,0 +1,53 @@
+(function (env) {
+    "use strict";
+
+    env.ddg_spice_glassdoor = function(api_result) {
+
+        if (!api_result || !api_result.response.employers || api_result.response.employers.length === 0) {
+            return Spice.failed("glassdoor");
+        }
+        
+        var query = DDG.get_query().replace('company','');
```

Comment:
Please could this also be updated from `company` to `glassdoor`, to match the trigger.